hw3/hw3_t02.py

```diff
@@ -0,0 +1,24 @@
+"""
+Here's a not very efficient calculation function that calculates something important.
+
+Calculate total sum of slow_calculate() of all numbers starting from 0 to 500.
+ Calculation time should not take more than a minute. Use functional capabilities of
+multiprocessing module. You are not allowed to modify slow_calculate function.
+"""
+
+import hashlib
+import random
+import struct
+import time
+from multiprocessing import Pool
+
+
+def slow_calculate(value: int) -> int:
+    time.sleep(random.randint(1, 3))  # noqa: S311
+    data = hashlib.md5(str(value).encode()).digest()  # noqa: S303
+    return sum(struct.unpack("<" + "B" * len(data), data))
+
+
+def slow_calc_500_numbers() -> list:
+    with Pool(processes=50) as pool:
```

Comment:
Why have you chosen 50?

Reply:
50 works well under 30 seconds, 10 is too slow, 100 doesn't finish at all.

Reply:
> 100 doesn't finish at all

What do you mean?   They are freezes without any message?

Reply:
No messages, no errors, just endless running.